examples/huggingface-transformers/pipelines.py

```diff
@@ -970,21 +978,32 @@ def _create_model(
             onnx_path, batch_size=1, num_cores=num_cores, num_sockets=num_sockets
         )
     elif engine_type == ORT_ENGINE:
-        sess_options = SessionOptions()
+        _validate_ort_import()
+        sess_options = onnxruntime.SessionOptions()
         if num_cores is not None:
             sess_options.intra_op_num_threads = num_cores
         sess_options.log_severity_level = 3
-        sess_options.graph_optimization_level = GraphOptimizationLevel.ORT_ENABLE_ALL
+        sess_options.graph_optimization_level = (
+            onnxruntime.GraphOptimizationLevel.ORT_ENABLE_ALL
+        )
 
-        model = InferenceSession(onnx_path, sess_options=sess_options)
+        model = onnxruntime.InferenceSession(onnx_path, sess_options=sess_options)
 
     return model, input_names
 
 
+def _validate_ort_import():
+    if ort_import_error is not None:
+        raise ImportError(
+            "An exception occurred when importing onxxruntime. Please verify that "
+            "onnxruntime is installed in order to use the onnxruntime inference engine"
+        )
+
+
 def _validate_transformers_import():
     if transformers_import_error is not None:
         raise ImportError(
-            "An exception occured when importing from the transformers library. "
+            "An exception occurred when importing from the transformers library. "
```

Comment:
Same thing, can we print out the error for better debugging along with the message